neqo-transport/src/connection.rs

```diff
@@ -852,8 +877,13 @@ impl Connection {
                 State::Handshaking | State::Connected => {
                     if !self.is_valid_cid(&hdr.dcid) {
                         qinfo!([self], "Ignoring packet with CID {:?}", hdr.dcid);
+                        self.stats.dropped_rx += 1;
                         return Ok(frames);
                     }
+                    if self.role == Role::Server && matches!(hdr.tipe, PacketType::Handshake {}) {
```

Comment:
```suggestion
                    if self.role == Role::Server && matches!(hdr.tipe, PacketType::Handshake {..}) {
```

Don't you need `{..}` to match a complex enum arm?

Reply:
it is funny that cargo is not complaining about this.